clients/mobile/src/components/inputElement/DateTimePicker.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useState } from 'react'
+import { Text, TouchableOpacity, View, StyleSheet } from 'react-native'
+import DateTimePicker from '@react-native-community/datetimepicker'
+import { formatteDate } from '../../utility/formatte'
+import { useTheme } from '../../setup/theme/hooks/useTheme'
+import { Theme } from '../../setup/theme'
+import { commonStyles } from './commonStyles'
+
+interface DateTimePickerComponentProps {
```

Comment:
if you're not extending, why use interface

Reply:
We are using interfaces for objects and `type` for simple data types like urgencyLevel = 'urgent' | 'regular'